### PR TITLE
Fix the macos Unit test - addon signature

### PR DESCRIPTION
**Pull request by bakulf**

### CODE REVIEW

**Review by brizental — APPROVED** [review]

File: src/models/feature.h [hunk 1]
```diff
@@ -160,6 +160,10 @@ class Feature : public QObject {
   bool m_released = false;
   // Is true if the Feature was released in this Version
   bool m_new = false;
+
+#ifdef UNIT_TEST
+  friend class TestAddonIndex;
+#endif
```
Comment (brizental) [inline src/models/feature.h:166]: Question: why is this necessary?

Comment (bakulf) [reply src/models/feature.h:166]: because maybeFlipOnOff is private.
